File: codeforces/648_Div2/a.py
for _ in range(int(input())):
    n,m = map(int, input().split())
    l = []
    colb = [False]*m
    rowb = [False]*n
    row = 0
    col = 0
    for i in range(n):
        l.append(tuple(map(int,input().split())))
        for j in range(m):
            if(l[i][j] == 1):
                colb[j] = 1
                rowb[i] = 1
    for i in range(n):
        if(rowb[i]==0):
            row+=1
    for i in range(m):
        if(colb[i]==0):
            col+=1
    row = min(row, col)
    if(row&1):
        print("Ashish")
    else:
        print("Vivek")

File: codeforces/648_Div2/a.cpp
#include <iostream>
#include <vector>
using namespace std;
int main()
{
    int t;
    cin>>t;
    while(t--)
    {
        int col=0,row=0;
        int n,m;
        bool temp;
        cin>>n>>m;
        bool colb[m], rowb[n];
        for(int i=0;i<n;i++)
        {
            for(int j=0;j<m;j++)
            {
                cin>>temp;
                if(temp)
                {
                    colb[j] = 1;
                    rowb[i] = 1;
                }
            }
        }
        for(int i=0;i<n;i++)
            if(!rowb[i])
                row++;
        for(int i=0;i<m;i++)
            if(!colb[i])
                col++;
        if(row>col)
            row = col;
        if(row&1)
            cout<<"Ashish\n";
        else
            cout<<"Vivek\n";
    }
}
